opus/application/static_media/js/browse.js

```diff
@@ -635,12 +638,13 @@ var o_browse = {
         let elem = $(`#${opus.prefs.view} .thumbnail-container[data-obs="${value}"]`);
         let startObsLabel = o_browse.getStartObsLabel();
 
+        // Update obsNum in infiniteScroll instances, and this obsNum is the first item in current page. (will be used to set scrollbar position in renderGalleryAndTable).
```

Comment:
Please break the long comments across multiple lines so they don't go past the edge of a reasonable browser width.